app/models/cinstance.rb

```diff
@@ -172,7 +172,9 @@ def self.by_service(service)
     if service == :all || service.blank?
       all
     else
-      where{ plan_id.in( my{Plan.issued_by(service).select(:id)} ) }
+      # NOTE: `reorder(nil)` resets the order set in the Plan's default scope,
+      # because ORDER BY in subqueries fails in Oracle with 'ORA-00907: missing right parenthesis'.
+      where { plan_id.in( my { Plan.issued_by(service).select(:id).reorder(nil)} ) }
```

Comment:
I'm wondering if it would be better to just remove the default scope (order by created_at) on Plan model, and add order explicitly wherever needed... :thinking:

Reply:
probably, but keeping `reorder(nil)` here is also better to avoid unnecessary ordering of the IDs. The order of plan ids here shouldn't matter

Reply:
If I understood this correctly, you want to remove the ordered default scope because that's not compatible with Oracle, is that right? in that case I think its a good idea to remove it.